src/applications/gi/reducers/calculator.js

```diff
@@ -310,7 +310,7 @@ export default function(state = INITIAL_STATE, action) {
         } else {
           giBillBenefit =
             action.payload.AVGDODBAH &&
-            action.payload.AVGDODBAH < action.payload.AVGBAH
+            action.payload.AVGDODBAH < action.payload.AVGVABAH
```

Comment:
it doesn't look like this behavior is gated- we have a `calculator.old.js` for the selector but not the reducer. 

`FETCH_PROFILE_SUCCEEDED` only adds `AVGVABAH` on staging

Reply:
If you look at line 301 you will see the production check that wraps this code.
`if (!environment.isProduction())`

If need be, I can create a new file and move the production flag check into `reducers/index.js`.

Reply:
I should have look further up. Thanks for pointing that out.